cpp/dolfin/fem/FormIntegrals.h

```diff
@@ -10,6 +10,7 @@
 #include <dolfin/common/types.h>
 #include <functional>
 #include <memory>
+#include <petscis.h>
```

Comment:
Is the right header to include? Is it just to get `PetscScalar`?

Reply:
No. It was meant to be `petscsys.h`. But in fact it's not really necessary.
We already get `PetscScalar` by importing `dolfin/common/types.h`.